### DISCUSSION TITLE
Dexcom One transmitter connectivity

### DISCUSSION THREAD

**Discussion by reyalex1** [NightscoutFoundation/xDrip#2714] [Q&A]
Hi, I've successfully connected to a Dexcom One transmitter using an old Samsung J5 but unable to do so using my regular phone - Google Pixel 6. The transmitter starts with a C. 

Both phones using the same version of xdrip and same settings as posted in the xdrip Facebook group by Navid Fo. Photo attached is the classic status page on the J5. 

Is there a possible issue with Pixel 6 comparability? I note on the Android APS Facebook group that some users report long connecting periods with this phone when using an Omnipod Dash (connection made via Bluetooth), so wondering if it's something to do with Pixel 6 and Bluetooth.

Separate question, if I change the hardware data source from Libre to Dexcom during an active libre sensor, will this stop the sensor or will it continue to work if I then revert back to my libre settings. Asking as I would like to test the Dexcom transmitter again now I know that it works for sure on another phone.
![inbound660480041447665315](https://user-images.githubusercontent.com/85752598/222467538-5c2515f6-4ded-4dda-a931-23f72f1ae2be.jpg)

**Comment by Navid200** [comment]
Samsung J5 is Android 6, right?
Pixel 6 is Android 13.
The xDrip settings are not the same.  https://navid200.github.io/xDrip/docs/G6-Recommended-Settings.html

Can you clarify what you mean when you say both phones use the same settings.

**Reply by reyalex1** [reply]
My mistake, settings are not the same. I followed the guide for Android 9 on the Samsung J5 (phone has 9 installed not 6) whereas the pixel 6 is Android 13 and followed the guide for Android 10 or newer with that.

**Comment by Navid200** [comment]
@psonnera Would you please help with the question about switching from and back to Libre?

**Reply by psonnera** [reply]
@Navid200 I don't really know what will happen indeed... @reyalex1 I wouldn't be too worried though as starting a Libre sensor can be done multiple times since xDrip+ doesn't start neither stop Libre sensors. As for the Dexcom sensor if you don't explicitly stop the sensor there's no reason anything bad will happen. What I wonder is if you'll be required to start or not the already started sensor or if xDrip+ will see a different sensor and will anyway reset all calibrations.

**Reply by reyalex1** [reply]
@Navid200 @psonnera  thank you both for the responses. I've uninstalled xdrip and reinstalled, checked all settings, and happy to report that everything is now working!
